ts-microservices/test/happy_path.test.ts

```diff
@@ -0,0 +1,35 @@
+import {expect} from '@jest/globals';
+
+const testUserName = "timmy"
+const axios = require('axios');
+const apiEndpoint = process.env['API_ENDPOINT']
```

Comment:
Where's this get set from?

Reply:
within our GH Action. We do this so that we can also set it locally to localhost or pulumi output